### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Buttons-Screenshot (47)

### CODE REVIEW

File: Components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    background-color: var(--primary-color);
+}
+
+.primary-btn:hover{
+    background-color: var(--dark-primary-color);
+}
+
+.primary-btn-outline{
+    color:var(--primary-color);
+ border:1px solid var(--primary-color);
```
Comment (junaidshaikh-js) [inline Components/buttons.css]: It is better to use prettier to format the code for consistent look

File: Components/buttons.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    padding-right: 0.6rem;
+}
+.secondary-btn-text-icon:hover{
+    background-color: #686c6e; 
+}
+
+/* Colored Buttons */
+
+.success-btn{
+    /* background-color: rgb(9, 155, 9) ; */
```
Comment (junaidshaikh-js) [inline Components/buttons.css]: commented code should be removed

**Review by junaidshaikh-js — COMMENTED** [review]
I saw many empty lines between the code block, you can remove them.

File: Components/buttons.css [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    background-color:var(--primary-color);
+    color:var(--complementary-text-color);
+}
+.primary-btn-text-icon .btn-icon{
+    color:var(--complementary-text-color);
+    padding-right: 0.6rem;
+}
+.primary-btn-text-icon:hover{
+    background-color: var(--dark-primary-color);
+
```
Comment (ritik-patel05) [inline Components/buttons.css]: remove empty space

File: Components/buttons.css [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+    color:inherit;
+}
+
+.link:hover{
+    color:var(--primary-color);
+}
+
+.link:focus{
+    border:2px dashed var(--primary-color);
+}
```
Comment (ritik-patel05) [inline Components/buttons.css]: Do add a new line at the end of the file. Reference:[link](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

File: Components/buttons.css [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+    background-color: #686c6e; 
+}
+
+/* Colored Buttons */
+
+.success-btn{
+    /* background-color: rgb(9, 155, 9) ; */
+    background-color: var(--success-color) ;
+    color:var(--complementary-text-color);
+
```
Comment (ritik-patel05) [inline Components/buttons.css]: remove empty space

File: Components/buttons.css [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+    border:4px solid var(--primary-color);
+    text-transform: uppercase;
+    letter-spacing: 0.2rem;
+    
+ }
+ 
+ .primary-btn-neon:hover{
+     box-shadow: 0 2px 0 var(--light-primary-color) inset, 0 2px 20px 0 var(--light-primary-color) ,
+                 0 2px 0 var(--light-primary-color)  inset, 0 2px 20px 0 var(--light-primary-color) ;
+                 text-shadow: 0 0 2px var(--light-primary-color) ,
```
Comment (ritik-patel05) [inline Components/buttons.css]: format

File: Components/buttons.css [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+}
+
+/* Neon buttons */
+
+.primary-btn-neon{
+    font-weight: 400;
+    border:4px solid var(--primary-color);
+    text-transform: uppercase;
+    letter-spacing: 0.2rem;
+
```
Comment (ritik-patel05) [inline Components/buttons.css]: remove empty space

File: Components/buttons.css [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+/* Disabled Button */
+ .btn-disabled{
+     opacity: 0.7;
+     pointer-events: none;
+ }
+
+/* CTA sizes */
+ .large-btn{
+
```
Comment (ritik-patel05) [inline Components/buttons.css]: remove empty space

File: Components/buttons.css [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+/* CTA sizes */
+ .large-btn{
+    
+     font-size: 1.2rem;
+     padding:0.7rem 1.4rem;
+ }
+
+ .small-btn{
+     font-size: 0.85rem;
+    padding:0.3rem 0.7rem;
```
Comment (ritik-patel05) [inline Components/buttons.css]: format

File: Components/buttons.css [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+    max-height: 25px;
```
Comment (ritik-patel05) [inline Components/buttons.css]: format

File: Components/buttons.css [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+
```
Comment (ritik-patel05) [inline Components/buttons.css]: remove empty space

**Review by ritik-patel05 — COMMENTED** [review]
- Use consistent spacing Tabs/Space 4/2 units.
- Use variables for commonly used spacing units, so that in the future it helps in making decisions easily and be consistent rather than typing random values.

File: Components/buttons.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/buttons.css]: used the prettier extension

File: Components/buttons.css (hunk 2 above)
Comment (Radheshyam1234) [reply Components/buttons.css]: Removed commented code

File: Components/buttons.css (hunk 8 above)
Comment (Radheshyam1234) [reply Components/buttons.css]: done

File: Components/buttons.css (hunk 9 above)
Comment (Radheshyam1234) [reply Components/buttons.css]: I have used prettier now

**Pull request merged**